### PR TITLE
update publish script

### PR DESCRIPTION
**Pull request by charlierudolph**
@kevgo @hugobho does docker need specific publishing rights?

I want to move publishing to docker / npm to travisci in the future so we don't need to hand out permissions.

### CODE REVIEW

File: go/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM golang:1.8
+
+WORKDIR /go/src/github.com/Originate/exocom/go/
+COPY . .
+
+WORKDIR exocom
+RUN go install
+RUN rm -rf /go/src/
+
+CMD /go/bin/exocom
```
Comment (charlierudolph) [inline go/Dockerfile]: Copying the entire go folder to get the vendor and shared (util / struct) directories. Build the binary, then remove all source to make the docker image smaller. Finally give it the default command as bin/exocom and thus we don't need to supply a command on exosphere to run exocom

Comment (kevgo) [inline go/Dockerfile]: What is this Dockerfile for? If for shipping Exocom, why can't we simply copy only the exocom binary into an empty Docker container? Or maybe one based on Busybox or Alpine Linux?

Comment (charlierudolph) [inline go/Dockerfile]: That should work. I'm just not sure what version we need for the docker image and this way it build it on the machine it runs.

**Comment by kevgo** [comment]
If we compile Exocom inside a Docker image, we should be able to simply
take the generated binary and put it into an empty Docker image.

**Comment by charlierudolph** [comment]
This is now updated with the simpler publish process

**Pull request merged**